src/CSF.SRDashboard.Client/Components/SeafarerDetails.razor.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Components;
+using CSF.SRDashboard.Client.Models;
+
+namespace CSF.SRDashboard.Client.Components
+{
+    public partial class SeafarerDetails
+    {
+        [Parameter]
+        public SeafarerDetailsModel Data { get; set; }
```

Comment:
```suggestion
        public SeafarerDetailsModel Model { get; set; }
```
To be consistent with `CSF.SRDashboard.Client.Components.RequestDetail`

Reply:
Good point.

Reply:
Changed